config/checkstyle.xml

```diff
@@ -9,313 +9,303 @@ page at http://checkstyle.sourceforge.net/config.html -->
 
 <module name="Checker">
 
-
-    <module name="FileTabCharacter">
-        <!-- Checks that there are no tab characters in the file.
+  <module name="FileTabCharacter">
+    <!-- Checks that there are no tab characters in the file.
         -->
+  </module>
+
+  <module name="NewlineAtEndOfFile"/>
+
+  <module name="RegexpSingleline">
+    <!-- Checks that FIXME is not used in comments.  TODO is preferred.
+      -->
+    <property name="format" value="((//.*)|(\*.*))FIXME"/>
+    <property name="message" value='TODO is preferred to FIXME.  e.g. "TODO (johndoe): Refactor when v2 is released."'/>
+  </module>
+
+  <module name="RegexpSingleline">
+    <!-- Checks that TODOs are named.  (Actually, just that they are followed
+         by an open paren.)
+    -->
+    <property name="format" value="((//.*)|(\*.*))TODO [^(]"/>
+    <property name="message" value='All TODOs should be named.  e.g. "TODO (johndoe): Refactor when v2 is released."'/>
+  </module>
+
+  <module name="RegexpSingleline">
+    <!-- Check for trailing whitespace. -->
+    <!-- \s matches whitespace character, $ matches end of line. -->
+    <property name="format" value="\s+$"/>
```

Comment:
Can we add `<property name="message" value='Please remove trailing whitespace.'/>` here?